components/color-picker/private/custom-color.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import CustomColorForm from './custom-color-form';
+import HsvColor from './hsv-color';
+
+// using state to manage everything in here because multiple HSV coordinates
```

Comment:
Generally, it's good to make things controlled, but I'm OK with this since it's in a dialog and isn't really a selection until done is pressed of the dialog is closed.